gcp/appengine/frontend3/src/templates/vulnerability.html

```diff
@@ -62,10 +62,43 @@ <h1 class="title">{{ vulnerability.id }}</h1>
           <dd>{{ vulnerability.published }}</dd>
           <dt>Modified</dt>
           <dd>{{ vulnerability.modified }}</dd>
+          {%- if vulnerability.severity -%}
+          <dt>Severity</dt>
+          <dd>
+            <ul class="aliases">
+              {% for item in vulnerability.severity -%}
+              <li>{{ item.type }} - {{ item.score }}</li>
```

Comment:
Can we render this with a monospaced font instead?

Reply:
updated [e60cc14](https://github.com/google/osv.dev/pull/2132/commits/e60cc14805fc17f867331bd58a229b7b950a863b)

You might verify this change via `GHSA-2c8c-84w2-j38j`. However, I noticed there are some bugs (e.g. `CVE-2024-22188`) that although the json includes the `severity` details, it's not recorded in the db. So this information will not be exposed via osv.dev.

sample output:

<img width="976" alt="Screenshot 2024-05-14 at 3 14 30 PM" src="https://github.com/google/osv.dev/assets/73332835/ce34bdae-0765-4429-aebc-d58395bad550">